ansible/roles/common/tasks/main.yml

```diff
@@ -50,6 +51,17 @@
     name: supervisor
     enabled: yes
 
+- name: Permit password sshd auth
+  lineinfile:
+    path: /etc/ssh/sshd_config
+    regexp: "^[#]?PasswordAuthentication "
```

Comment:
You can simplify the regex: `^#?PasswordAuthentication `.

Reply:
Yeah, I'm concerned it's harder to mentally parse the simpler version, but ok.